modules/kubernetes-addons/main.tf

```diff
@@ -333,6 +333,15 @@ module "opentelemetry_operator" {
   source        = "./opentelemetry-operator"
   helm_config   = var.opentelemetry_operator_helm_config
   addon_context = local.addon_context
+
+  # Amazon EKS ADOT add
```

Comment:
Don't forget we'll need to support both the EKS ADOT route as well as the current self-managed otel operator. See [here](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/639/files#diff-ae264c430a0512ec2b96032bc8a245bb09a2da086fc5edd677c1d31fb25dca71R324-R339) for reference

Reply:
Yes! just updated this